cli/lib/kontena/cli/volume_command.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'volumes/create_command'
+require_relative 'volumes/remove_command'
+require_relative 'volumes/list_command'
+
+class Kontena::Cli::VolumeCommand < Kontena::Command
+
+  subcommand "create", "Create a managed volume", Kontena::Cli::Volume::CreateCommand
```

Comment:
The lazy subcommand loader thing is in master now, so these should also be:

```
subcommand "create", "Create a managed volume", load_subcommand('volumes/create_command')
```

(also remove the requires above)